panoptes/db.config

```diff
@@ -0,0 +1,19 @@
+{"Database":
```

Comment:
For usability reasons, I would very much prefer to have a clear indication on the file format that we use. Something like db_config.json....

Reply:
done file name change to db_config.json
done format